firmware/config/boards/me7_pnp/STM32F76xxI.ld

```diff
@@ -0,0 +1,134 @@
+/*

```

Comment:
is this fine different from the file in the 767 folder? We do not need two copies of the same file.

Reply:
Please update the pull request or create a new one

Reply:
Would be nice to have them separated, so the files can be edited / hardcoded.
By that I can easier keep track of what's happening and where.

Reply:
@960 please respond to "how is this file different from the file in the 767 folder?"
if it's same, it could not be added again.

Reply:
@960 
@960 please respond to "how is this file different from the file in the 767 folder?"
if it's same, it could not be added again.

Reply:
It's copied from the Nucleo folder. 
Did not know they couldnt be the same.
But I dont need a separate linker script.